app/helpers/zendesk_service_helper.rb

```diff
@@ -99,10 +99,22 @@ def create_ticket(subject:, requester_id:, group_id:, body:, fields: {})
   def append_file_to_ticket(ticket_id:, filename:, file:, comment: "", fields: {})
     raise MissingTicketIdError if ticket_id.blank?
 
+    append_multiple_files_to_ticket(
+      ticket_id: ticket_id,
+      file_list: [{ filename: filename, file: file }],
+      comment: comment,
+      fields: fields
+    )
+  end
+
+  # file_list is an Array of Hashes with keys :file and :filename.
```

Comment:
for this case, would it make sense to just pull the filename from each File object? I'm not sure if that's possible by default, but it seems like a reasonable API to expect.